test_evm_eth_compliance/src/common/bits.rs

```diff
@@ -0,0 +1,289 @@
+use derive_more::{AsRef, Deref};
+use fixed_hash::{construct_fixed_hash, impl_fixed_hash_conversions};
+
+construct_fixed_hash! {
```

Comment:
If these types are only used in the model, I would consider moving all of this into the `model` package.